### DISCUSSION TITLE
I Have No Trade Button

### DISCUSSION THREAD

**Discussion by TikSmous** [Chia-Network/chia-blockchain#4817] [Support]
Hi there

I recently started farming and noticed that I don't have a trade button on my user interface

Is it because Chia is not yet tradeable in South Africa(my location)?

If so ,when will it become tradeable and what are my options at this moment?

**Comment by moshenskyDV** [comment]
Could you please attach screenshot of "Wallet" tab?

**Reply by TikSmous** [reply]
![image](https://user-images.githubusercontent.com/84143532/118332977-3433a680-b50b-11eb-8d90-477758079d06.png)

**Reply by moshenskyDV** [reply]
I see possibility to send chia and to receive chia (as it should be)? So everything is fine, is it?

If you want to swap chia to usdt, btc, eth or other coins - you should do it on marketplace. You can find list of available markets here: [https://coinmarketcap.com/currencies/chia-network/markets/](https://coinmarketcap.com/currencies/chia-network/markets/)
Cheers!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
